benchmarks/dynamo/common.py

```diff
@@ -1970,33 +1970,6 @@ def get_dynamo_stats():
     )
 
 
-def maybe_fresh_cache(fn, is_cold_start):
```

Comment:
I moved this "higher up" in the call stack because I want to persist the fresh cache across two runs